src/main/java/Main.java

```diff
@@ -0,0 +1,49 @@
+import de.neuefische.Model.Product;
+import de.neuefische.OrderRepo.Order;
+import de.neuefische.OrderRepo.OrderRepo;
+import de.neuefische.ProductRepo.ProductDB;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        OrderRepo listOfOrders = new OrderRepo();
+
+
+
+        Product screws = new Product(1,"Screws");
+        Product nails = new Product(2,"Nails");
+        Product mouse = new Product(3, "Wireless Mouse");
+        Product keyboard = new Product(4, "Keyboard");
+        Product monitor = new Product(5, "Monitor");
```

Comment:
Gute Namensgebung! 

product1
product2
product3

wäre auch möglich, aber in Fällen bei denen es gut zu wissen wäre was die Objekte sind, ist deine Lösung natürlich top!

Reply:
Hey Sergej,
ja, Anfang hatte ich die Products nach ihrem Inhalt benannt, fand ich dann im weiteren Verlauf aber einfacher sie zu nummerieren... Das finden von Variablennamen empfinde ich ohnehin immer als grosse Herausforderung XD

Reply:
Wie gesagt, ist (in dem Fall) beides möglich/ akzeptierbar!

> Das finden von Variablennamen empfinde ich ohnehin immer als grosse Herausforderung XD

True! :100:  _Das_ ist die eigentliche Kunst der Softwareentwicklung Falco! :joy: